### PR TITLE
Dev/normalization/proxy parsers

### PR DESCRIPTION
**Pull request by oshezaf**

### CODE REVIEW

File: Parsers/ASimWebSession/Parsers/imWebSession.yaml [hunk 1]
```diff
@@ -13,23 +13,29 @@ References:
...
     starttime:datetime=datetime(null), 
     endtime:datetime=datetime(null),
-    srcipaddr_has_any_ipv4_prefix:dynamic=dynamic([]),
+    srcipaddr_has_any_prefix:dynamic=dynamic([]),
     url_has_any:dynamic=dynamic([]), 
     httpuseragent_has_any:dynamic=dynamic([]), 
-    eventresultdetails_in:dynamic=dynamic([])
+    eventresultdetails_in:dynamic=dynamic([]),
+    eventresult:string='*',
+    disabled:bool = false)
```
Comment (yaronMSFT) [inline Parsers/ASimWebSession/Parsers/imWebSession.yaml:30]: does the metaparser need disabled?

File: Parsers/ASimWebSession/Parsers/vimWebSessionSquidProxy.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (
+    starttime:datetime=datetime(null), 
+    endtime:datetime=datetime(null),
+    srcipaddr_has_any_prefix:dynamic=dynamic([]), 
+    url_has_any:dynamic=dynamic([]),
+    httpuseragent_has_any:dynamic=dynamic([]),
+    eventresultdetails_in:dynamic=dynamic([]),
```
Comment (yaronMSFT) [inline Parsers/ASimWebSession/Parsers/vimWebSessionSquidProxy.yaml:54]: this parameter is not defined in the YAML

File: Parsers/ASimWebSession/Parsers/imWebSession.yaml [hunk 3]
```diff
@@ -13,23 +13,29 @@ References:
...
+    vimWebSessionzScalerZIA (starttime, endtime, srcipaddr_has_any_prefix, url_has_any, httpuseragent_has_any, eventresultdetails_in, eventresult, vimBuiltInDisabled or ('vimWebSessionzScalerZIA' in (DisabledParsers)))
   };
-  parser (starttime, endtime, srcipaddr_has_any, url_has_any, httpuseragent_has_any, eventresultdetails_in)
+  parser (starttime, endtime, srcipaddr_has_any, url_has_any, httpuseragent_has_any, eventresultdetails_in, eventresult)
 
-Parsers:
\ No newline at end of file
+Parsers:
+  - vimWebSesssionEmpty
+  - vimWebSessionSquidProxy
```
Comment (yaronMSFT) [inline Parsers/ASimWebSession/Parsers/imWebSession.yaml:41]: vimWebSessionzScalerZIA

File: Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionzScalerZIA.yaml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  ];
+  let parser=(disabled:bool=false){
+  CommonSecurityLog | where not(disabled)
+  | where DeviceVendor == "Zscaler"
+  | where DeviceProduct == "NSSFWlog"
+  // Event fields
+  | extend 
+    EventCount=DeviceCustomNumber1, 
+    EventStartTime=TimeGenerated, 
+    EventVendor = "zScaler",
```
[the remark this answers is not in the document]
Comment (oshezaf) [reply Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionzScalerZIA.yaml:45]: I prefer to ensure consistency. Even if just a mistake in the source format string...

**Comment by oshezaf** [comment]
@YaronFruchtmann : can you review following my updates? thanks!

**Pull request merged**